packages/project-access/src/project/ui5-config.ts

```diff
@@ -28,15 +28,15 @@ export async function getWebappPath(projectRoot: string, memFs?: Editor): Promis
 /**
  * Checks if UI5 config yaml file exists and returns its content.
  *
- * @param basePath - path to project root
+ * @param projectRoot - path to project root
```

Comment:
changed from `basePath` to `projectRoot` for consistency, no code semantic change